GrandCentralBoard/Widgets/Watch/WatchWidget.swift

```diff
@@ -6,9 +6,7 @@
 import UIKit
 import GrandCentralBoardCore
 
-
-private let slideshowInterval = 10.0
-private let secondsInDay: NSTimeInterval = 3600
+private let secondsInAnHour: NSTimeInterval = 3600
```

Comment:
Thanks for the references. 👍  The problem is that our project don't use any of these. If you want to change our agreement (so that name starts with small letter) please discuss that on Slack.